ads/kixer.js

```diff
@@ -45,6 +46,17 @@ export function kixer(global, data) {
   };
   d.addEventListener('load', kxload, false);
 
+  const unlisten = global.context.observeIntersection(function(changes) {
+    changes.forEach(function(c) {
+      if (c.intersectionRect.height > 0) {
+        if (typeof __kx_viewability.process_locked === 'function') {
+          __kx_viewability.process_locked(data.adslot);
+        }
+        unlisten();
```

Comment:
Hi Lannka, thanks for the feedback. 
This should be called only once now, although it wouldn't affect if we get multiple calls.